src/lib/components/Swap/Settings/MaxSlippageSelect.tsx

```diff
@@ -97,10 +107,16 @@ const Warning = memo(function Warning({ state, showTooltip }: { state: WarningSt
   )
 })
 
-export default function MaxSlippageSelect() {
+interface MaxSlippageSelectHandle {
+  reset: () => void
+}
+
+const MaxSlippageSelect = forwardRef<MaxSlippageSelectHandle>(function MaxSlippageSelect(_, ref) {
```

Comment:
variable shadowing here intentional?

Reply:
I really want to learn about this and also contribute, please if I can assist let me know what to do

Reply:
this is intentional - lint requires me to name the function, but the function and the wrapped function are functionally the same so I don't want to introduce a new name.

Reply:
Hi @Rds2008ca-libellule, glad to see you want to contribute!

We maintain [a list of "good first issues" on GitHub](https://github.com/Uniswap/interface/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22). That is the best place to get started contributing.

Reply:
Ok thanks